client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -181,7 +181,7 @@ describe( 'EditorGroundControl', function() {
 				/>
 			).instance();
 
-			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Publish' );
+			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Update' );
```

Comment:
So I think that is working as expected right now in this branch - but the test case doesn't align properly:

![past-dated](https://cloud.githubusercontent.com/assets/22080/14793913/2db38a8a-0ad6-11e6-8472-3281ec462d65.gif)